wordpress-mobile/WordPress-iOS
# Feature/9854 stock photo export media

Fixes part of #8959 

This PR implement export of the media asset chosen by the user in Stock Photos.

![export_](https://user-images.githubusercontent.com/9772967/38202636-cbbed792-3672-11e8-86d1-637ed703a517.gif)

To test:

1. Start a new post or edit an existing one
2. Tap the + button in the lower Format Bar.
3. Tap the ellipsis in the Format Bar.
4. Select "Free Photos Library".
5. Insert `cat` in the search bar (or `kitten`).
6. Select one or more cat (or kitten).
7. Press `Select X` in the top right corner.
8. The selected cat(s) or kitten(s) should load in the editor.
9. They should also be added to your WordPress Media Library.
